apphome/apphome.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
where is this used?

Reply:
AH sorry it should not be part of this PR, but the next one ^^"